src/app/globalStatusBar/staticStub/public.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @name publicParse
+ * @desc Parse contest public information
+ * @param {Function} func - Function to execute after the promise resolves.
+ * @author minhducsun2002
+ */
+
+async function publicParse(func) {
+    return fetch(
+        `http://${window.location.hostname}:${window.location.port}/api/info`
```

Comment:
Shorten this to "/api/info"

Reply:
Addressed in https://github.com/vnma0/hestia/pull/19/commits/84c0282afababde0e3ad70cf90345d947d22553c.